module/userProjects/helper.js

```diff
@@ -3911,7 +3963,7 @@ function _updateUserProfileBasedOnUserRoleInfo(userProfile, userRoleInformation)
                         "id" : locationIds
                     }
 
-                    let entityData = await userProfileService.locationSearch(locationQuery);
+                    let entityData = await userService.locationSearch(locationQuery);
```

Comment:
@Prajwal17Tunerlabs location search is called here . please check with @priyanka-TL how we planned to make changes with location search. We should be using a api from our entity-management service

Reply:
replaced locationService with entity-service